lib/tasks/memcached.rake

```diff
@@ -0,0 +1,19 @@
+namespace :memcached do
+
+ # For example
+ # WORKS="posted = 0" rake memcached:clear_work
+ #
+ desc "Clear memcached"
+ task :clear_work => :environment  do
```

Comment:
Er, and while I'm here, it might make more sense to call the task ":expire_work_blurbs", since there are a few different ways we cache works.